upstream.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = xxHash-0.8.0
+
+source_url = https://github.com/Cyan4973/xxHash/archive/v0.8.0.tar.gz
+source_filename = xxHash-0.8.0.tar.gz
+source_hash = 7054c3ebd169c97b64a92d7b994ab63c70dd53a06974f1f630ab782c28db0f4f
+
+[provides]
+dependency_names = libxxhash
+program_names = xxhsum
```

Comment:
So you are essentially saying that I should just remove the `[provide]` section for this contribution and future wrapdb contributions because it doesn't really make sense from a wrapdb context?
Sorry if I am not completely following.

Reply:
No... I'm saying the wrapdb documentation says for compatibility with the widest range of meson versions, you should use
```
[provide]
libxxhash = xxhash_dep

```
and the example of most wraps demonstrates doing it this way.

You could drop program_names though (does anyone even need the program at all?)

Reply:
`dependency_names = name` does NOT expose the dependency to lookups, but only tells meson "this wrap contains an override_dependency()". It is the override_dependency() that then exposes it for lookups.

`name = variable` exposes the dependency for lookups directly.

Reply:
I doubt anyone needs the program, but I just wanted to be complete in the port. I find it pretty annoying the port does not fully support the native build system. lz4 is one such example that I might try my hand at fixing after this.

Reply:
I'll just drop it for compat. If people complain then I can add it back.